### PR TITLE
Issue/extract components from home

### LINKED ISSUE #11: Show current sell price
The current sell price is a very important value, we will show it in ETH and USD.

It wont be represented in the graphic, but it is something that the user will want to see near it, maybe above the graphic in the center?

### LINKED ISSUE #16: Extract componentes
Currently `Home view` is very coupled. Adding the code to fetch the real values for `Price` and `Chart` to the `Home view` is not a good idea. 
Please, extract `Price` and  `Chart` to other components will be great.

### PR DESCRIPTION
**Pull request by skkr**
closes #16 
closes #11 
also made other improvements related to components

### CODE REVIEW

File: src/components/ChartPrice.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+import SellPriceContainer from './SellPriceContainer'
+
+// Modules
+var LineChart = require("react-chartjs").Line;
```
Comment (MatiasOS) [inline src/components/ChartPrice.js]: Can we use like `import { Line as LineChart } from 'react-chartjs';`?

File: src/components/ChartPriceContainer.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+
+import ChartPrice from './ChartPrice'
+
+export default class ChartPriceContainer extends Component {
+
+  static defaultProps =  {
```
Comment (MatiasOS) [inline src/components/ChartPriceContainer.js]: this defaultProps, must be in `ChartPrice`. Sorry for not having expressed it correctly when I explained

File: src/components/SellPriceContainer.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+
+import SellPrice from './SellPrice'
+
+export default class SellPriceContainer extends Component {
+
+  static defaultProps =  {
```
Comment (MatiasOS) [inline src/components/SellPriceContainer.js]: The same as above. This must be in `SellPrice`.

**Review by MatiasOS — CHANGES_REQUESTED** [review]

**Comment by skkr** [comment]
@MatiasOS please review it now

**Review by MatiasOS — APPROVED** [review]
:clap: :clap:  :clap:

**Pull request merged**